src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Sides must be greater than 0');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Can`t form a triangle');
```

Comment:
[CODE STYLE] - When throwing an error, provide a more detailed description in the error message to make it clear what the problem is. For example, instead of 'Sides must be greater than 0', you could say 'All triangle sides must be greater than 0 to form a valid triangle'. Similarly, instead of 'Can`t form a triangle', a more descriptive message would be 'The provided dimensions cannot form a valid triangle as they do not satisfy the triangle inequality theorem'.